Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml

```diff
@@ -132,7 +139,7 @@ args:
   - 'True'
   - 'False'
   required: false
-  secret: false
+  secret: true
```

Comment:
why it's secret?

Reply:
changed automatically, will undo this change